specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ContainerApps.json

```diff
@@ -640,6 +641,10 @@
             "modelAsString": true
           }
         },
+        "targetLabel": {
```

Comment:
Update existing example for this property

Reply:
addded in latest commit